benchmarks/benchmark.py

```diff
@@ -435,7 +435,7 @@ def run_benchmark():
 def modify_config_props_for_mms(pargs):
     shutil.copyfile(CONFIG_PROP_TEMPLATE, CONFIG_PROP)
     with open(CONFIG_PROP, 'a') as f:
-        f.write('\nnumber_of_netty_threads=32')
+        f.write('number_of_netty_threads=32')
```

Comment:
Just curious, what was the error here? How is the job_queue_size working fine with \n

Reply:
Here's the main issue with the extra \n : 

If on the current netty branch one runs the command : 
```
python benchmarks/benchmark.py -l 10 -s --model lstm_ptb -w 8 -v
```
This results into the following error inside : (Which I believe is due to a recent validation added in code)

```
docker exec mms_benchmark_cpu mxnet-model-server --start --mms-config /mxnet-model-server/benchmarks/config.properties
Traceback (most recent call last):
  File "/usr/local/bin/mxnet-model-server", line 11, in <module>
    load_entry_point('mxnet-model-server', 'console_scripts', 'mxnet-model-server')()
  File "/mxnet-model-server/mms/model_server.py", line 74, in start
    props = load_properties(args.mms_config)
  File "/mxnet-model-server/mms/model_server.py", line 121, in load_properties
    props[key] = pair[1]
IndexError: list index out of range
/usr/local/Cellar/jmeter/4.0/libexec/bin/jmeter -n -t /Users/ghaipiyu/Documents/git-repos/mxnet-model-server/benchmarks/jmx/textInputModelPlan.jmx -Jaggregate_rpt_pct1=50 -Jaggregate_rpt_pct2=90 -Jaggregate_rpt_pct3=99 -Jdata=/Users/ghaipiyu/Documents/git-repos/mxnet-model-server/benchmarks/lstm_ip.json -Jhostname=127.0.0.1 -Jjmeter.reportgenerator.overall_granularity=1000 -Jloops=10 -Jmanagement_port=8081 -Jmin_workers=8 -Jmodel_name=lstm_ptb -Jperfmon_file=/tmp/MMSBenchmark/out/latency/lstm_ptb/perfmon.csv -Jport=8080 -Jprotocol=http -Jrampup=5 -Jthreads=1 -Jurl=https://s3.amazonaws.com/model-server/models/lstm_ptb/lstm_ptb.model -l /tmp/MMSBenchmark/out/latency/lstm_ptb/output.jtl -j /tmp/MMSBenchmark/out/latency/lstm_ptb/jmeter.log -e -o /tmp/MMSBenchmark/out/latency/lstm_ptb/report
Creating summariser <summary>
Created the tree successfully using /Users/ghaipiyu/Documents/git-repos/mxnet-model-server/benchmarks/jmx/textInputModelPlan.jmx
Starting the test @ Mon Oct 08 21:25:59 PDT 2018 (1539059159433)
Waiting for possible Shutdown/StopTestNow/Heapdump message on port 4445
summary +      1 in 00:00:00 =    5.5/s Avg:    89 Min:    89 Max:    89 Err:     1 (100.00%) Active: 1 Started: 1 Finished: 0
summary +     12 in 00:00:00 =  101.7/s Avg:     4 Min:     2 Max:    18 Err:    12 (100.00%) Active: 0 Started: 1 Finished: 1
summary =     13 in 00:00:00 =   43.5/s Avg:    11 Min:     2 Max:    89 Err:    13 (100.00%)
Tidying up ...    @ Mon Oct 08 21:26:00 PDT 2018 (1539059160190)

```

I did not want to change the core logic around reading the config props file in code and hence did not change any checks there.

Reply:
We should fix the code in MMS. I will raise a PR for the fix.

Reply:
Cool. I will close this one then.